### PR TITLE
Set VLAN Priority on dhcp6c packets

### PR DESCRIPTION
**Pull request by marjohn56**
Add VLAN tag to outgoing dhcp6c packets. Option setting in WAN dhcp6c section.

### CODE REVIEW

File: src/etc/inc/filter.inc [hunk 1]
```diff
@@ -3100,8 +3100,9 @@ function filter_generate_user_rule($rule) {
 }
 
 function filter_rules_generate() {
-	global $config, $g, $FilterIflist, $time_based_rules, $GatewaysList, $tracker;
+	global $config, $g, $FilterIflist, $time_based_rules, $GatewaysList, $tracker, $vlanprio_values;
 
+	$wancfg = $config['interfaces']['wan'];
```
Comment (rbgarga) [inline src/etc/inc/filter.inc]: Why is it using `'wan'` interface hardcoded here?

Comment (marjohn56) [reply src/etc/inc/filter.inc]: Don't know... copy paste finger trouble perhaps.

It's Removed.

Comment (marjohn56) [reply src/etc/inc/filter.inc]: I had a muppet moment... try the update.

**Comment by marjohn56** [comment]
Removed fixed 'wan' and used the existing routine to get the info required.

**Comment by kwillers** [comment]
Finally back in France where this functionality is needed with Orange FTTP service 

Works like a charm

**Comment by marjohn56** [comment]
Good stuff.

We still need to get the modified dhcp6c client sorted too, that will finally end the problems with Orange France.

**Comment by kwillers** [comment]
Indeed if we can get the modified dhcp6c client into the main build along with the modified dhclient client then Orange fttp users in France will able to upgrade via the GUI and not need manually replace the files each time

The two modified clients are very stable as my firewall has been up without issues for 3 months

**Comment by marjohn56** [comment]
I have issues a PR for dhcp6c changes on hrs-allbsd/wide-dhcpv6

Add the ability to use RAW parameters in the config PR #8

**Comment by jim-p** [comment]
In an effort to improve the tracking of changes and bug/feature requests, we have decided to require an entry on the [pfSense Redmine issue tracker](https://redmine.pfsense.org) associated with every pull request, and likewise the Redmine entry should also have a link back to the pull request.

If you could, please associate this PR to a Redmine issue either by locating an existing issue at https://redmine.pfsense.org or by creating a new issue. Add a link to the Redmine issue that points to this PR, and also add a link on the PR to the Redmine issue.

For more information, see https://doc.pfsense.org/index.php/Submitting_a_Pull_Request_via_Github

Thanks!

**Comment by marjohn56** [comment]
Redmine 8200

**Comment by marjohn56** [comment]
Redmine #8200

**Comment by jim-p** [comment]
Redmine Issue: https://redmine.pfsense.org/issues/8200

**Pull request merged**